docs/sources/send-data/promtail/configuration.md

```diff
@@ -1951,6 +1951,12 @@ or [journald](https://docs.docker.com/config/containers/logging/journald/) loggi
 Note that the discovery will not pick up finished containers. That means
 Promtail will not scrape the remaining logs from finished containers after a restart.
 
+The Docker target correctly joins log segments if a long line was split into different frames by Docker.
+To avoid hypothetically unlimited line size and out-of-memory errors in Promtail, this target applies
+a default soft line size limit of 256 kiB. If the buffer increases above this size, then the line will
```

Comment:
can we just clarify that that 256k default comes from prometheus' default max line size config

Reply:
Assume you meant Loki as we discussed :) It's updated